hermes-console/static/js/console/topic/TopicRepository.js

```diff
@@ -20,7 +20,7 @@ repository.factory('TopicRepository', ['DiscoveryService', '$resource', '$locati
         }
 
         function wrapSchemaRemove(topic, promise) {
-            if (topicConfig.removeSchema && topic.contentType == 'AVRO') {
+            if (topicConfig.removeSchema && topic.contentType === 'AVRO') {
```

Comment:
I don't think so... unless topic.contentType would be object that resolves somehow to String 'AVRO'?